Library/Homebrew/keg_relocate.rb

```diff
@@ -16,9 +16,10 @@ def fix_dynamic_linkage
       link = file.readlink
       # Don't fix relative symlinks
       next unless link.absolute?
-      if link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
-        FileUtils.ln_sf(link.relative_path_from(file.parent), file)
-      end
+      next unless link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
```

Comment:
Can you make this two `next unless`s for line length and for stupid Mike who can't handle `unless ||`.

Last thing and otherwise looks 👍